code-challenges.js

```diff
@@ -22,47 +22,105 @@ const fruit4 = "kiwi"
+        return `${number} is above boiling point.`
+    } else if (number < boilingPoint){
+        return `${number} is below boiling point.`
+    } else if (number === boilingPoint){
+        return `${number} is at boiling point.`
+    } else return "Please enter a temperature."
+}
+
+console.log(isTemperatureBoiling(temperature1)) // Output: "42 is below boiling point."
 // Expected output: "42 is below boiling point"
 
-const temperature2 = 350
+console.log(isTemperatureBoiling(temperature2)) // Output: "350 is above boiling point."
 // Expected output: "350 is above boiling point"
 
-const temperature3 = 212
+console.log(isTemperatureBoiling(temperature3)) // Output: "212 is at boiling point."
 // Expected output: "212 is at boiling point"
 
+console.log(isTemperatureBoiling()) // Output: "Please enter a temperature."
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
 
+    // Create code that combines the two arrays into a new array utilizing <  array1.concat(array2)  >.
+    // Then using the new array, return the lenght of said array utilizing <  array3.length  >.
+
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+
+const padresCombinedWorldSeriesRuns = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+
+console.log(padresCombinedWorldSeriesRuns) // Output: [2, 5, 2, 2, 4, 6, 3, 5, 3]
+
+console.log(padresCombinedWorldSeriesRuns.length) // Output: 9
 // Expected output: 9
 
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
 
+    // Create code that utilizes <  string..split("").reverse().join("")  > to reverse the provided string.
+    // <  .split("")  > will separate the characters in the string and mutate it into an array.
+    // < .reverse()  > will reverse the order of the values in the array.
+    // <  .join("")  > will unite all the characters in the array and return a new string.
+
 const currentCohort = "Bravo 2023"
+
+const reversedCohort = (currentCohort.split("").reverse().join("")) // Output: "3202 ovarB"
+
+console.log(reversedCohort) // Output: "3202 ovarB"
 // Expected output: "3202 ovarB"
 
+
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Pseudo code:
+    // Utilizing the < .lastIndex() > to find the last index of a given value using the variables given.
+    // The code created works by either using the givenValue# or by entering the value on the array for with index we want to find.
 
 const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 const givenValue1 = 42
+
+console.log(myNumbers.lastIndexOf(givenValue1)) // Output: 7
 // Expected output: 7
 
 const givenValue2 = 10
+
+console.log(myNumbers.lastIndexOf(10)) // Output: 10
```

Comment:
Nice work! I like that you are experimenting and using the variable (givenValues1) and the data itself (10).